[#1890] Updates zero-body warning messaging

hcraigmiller opened the pull request:
Updates Zero-body warning messaging from "Zero length bodies. . ." to "Zero-length/diameter bodies. . ."

neilweinstock commented:
Man I wish we could reword "may not result in accurate simulations".  That's a very mushy passive construction.

Not urgent to fuss with it right now though.

hcraigmiller commented:
"Zero-length/diameter bodies may not accurately simulate"?

neilweinstock commented:
I would go with "may not simulate accurately".  This also applies to a number of other messages (THICK_FIN, JAGGED_EDGED_FIN, PARALLEL_FINS, TUBE_SEPARATION, TUBE_OVERLAP.  These all say the same thing using different words.

This is just a band-aid until one day, I hope, we can overhaul the whole warning system, which is wholly inadequate right now.  Consistent wording would be a nice little mini-improvement in the meantime.

hcraigmiller commented:
Done.

![Zero-body Warning](https://user-images.githubusercontent.com/68821492/207947591-a60aa493-3a15-4b6b-b7c1-cc389ae77c19.png)

neilweinstock commented:
Are you going to change the others in this PR, or should I start a new one?

SiboVG commented on core/resources/l10n/messages.properties:
```diff
@@ -1859,8 +1859,8 @@ Warning.RECOVERY_HIGH_SPEED = Recovery device deployment at high speed
 Warning.EVENT_AFTER_LANDING = Flight Event occurred after landing:  
-Warning.ZERO_LENGTH_BODY = Zero length bodies may not result in accurate simulations.
-Warning.ZERO_RADIUS_BODY = Zero length bodies may not result in accurate simulations.
+Warning.ZERO_LENGTH_BODY = Zero-length/diameter bodies may not simulate accurately.
```
I would remove the redundancy and keep only one value, e.g. rename it to e.g. `Warning.ZERO_VOLUME_BODY` and then change `FinSetCalc.java` line 104 to:

```Java
if ((bodyLength < EPSILON) || (bodyRadius < EPSILON)) {
	// Add warnings: Phantom Body
	warnings.add(Warning.ZERO_VOLUME_BODY);
} else if( (0 < bodyRadius) && (thickness > bodyRadius / 2)){
```

hcraigmiller replied:
Warning.java also required a change, along with the translations (that will need to be verified/modified).

hcraigmiller commented:
The change in approach suggested by @SiboVG has been implemented.

neilweinstock commented:
Wait, now we're back to "may not result in accurate simulations"???

hcraigmiller commented:
My oversight, I reverted too far. Changed to "may not simulate accurately." I also changed the warning for tube separation and tube overlap to this phrase as well.

Thanks for catching that @neilweinstock.
